maintenance/wikia/createUser.php

```diff
@@ -43,7 +43,7 @@
 # Insert the account into the database
 $user->addToDatabase();
 $user->setEmail( $email );
-$user->setPassword( $password );
+$user->setPassword( $password, false );
 $user->confirmEmail();
 UserLoginHelper::removeNotConfirmedFlag( $user ); // this calls saveSettings();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS125)
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS139)